cmd/kueuectl/app/testing/fake.go

```diff
@@ -9,6 +9,8 @@ import (
 	"sigs.k8s.io/kueue/cmd/kueuectl/app/util"
 )
 
+const DefaultNamespace = "default"
```

Comment:
```suggestion
```

Could you use the metav1.DefaultNamespace instead of this?

https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#pkg-constants

Reply:
Good point. Thanks.